snapshots/devmapper/README.md

```diff
@@ -41,3 +41,136 @@ ctr run --snapshotter devmapper docker.io/library/hello-world:latest test
 
 The devicemapper snapshotter requires `dmsetup` (>= 1.02.110) command line tool to be installed and
 available on your computer. On Ubuntu, it can be installed with `apt-get install dmsetup` command.
+
+### How to setup device mapper thin-pool
+
+There are many ways how to configure a devmapper thin-pool depending on your requirements, disk configuration,
+and environment.
+
+On local dev environment you can utilize loopback devices. This type of configuration is simple and suits well for
+development and testing (please note that this configuration is slow and not recommended for production uses).
+Run the following script to create a thin-pool device:
+
+```bash
+#!/usr/bin/env bash
```

Comment:
nit: just `/bin/bash`

Reply:
My two cents. `/usr/bin/env bash` might be better because env will parse the PATH to get bash binary. Some distributions uses different path for bash :)